mmv1/products/privateca/api.yaml

```diff
@@ -328,6 +328,88 @@ objects:
                           item_type: Api::Type::Integer
                           description: |
                             An ObjectId specifies an object identifier (OID). These provide context and describe types in ASN.1 messages.
+              - !ruby/object:Api::Type::NestedObject
+                name: 'nameConstraints'
```

Comment:
Do you think if it makes sense to mark one of the subfields required, likely `critical`? This way, we can prevent user from sending empty block `name_constraints{}`, as it usually end up causing more problems.

Reply:
Can customer set a `required` boolean field to `false`?

Reply:
yeah, I believe they should be able to

Reply:
Done.